# File:"C:\Users\Yang\Documents\PTI\PSSE34\test-IEEE\IEEE9busSystem - fix bug\Init.py", generated on WED, MAY 16 2018  15:53, PSS(R)E release 34.03.02
psspy.case(r"""C:\Users\Yang\Documents\PTI\PSSE34\test-IEEE\1-39Bus-test\IEEE39_v33.sav""")
psspy.fnsl([0,0,0,1,1,0,99,0])
psspy.cong(0)
psspy.conl(0,1,1,[0,0],[ 100.0,0.0,0.0, 100.0])
psspy.conl(0,1,2,[0,0],[ 100.0,0.0,0.0, 100.0])
psspy.conl(0,1,3,[0,0],[ 100.0,0.0,0.0, 100.0])
psspy.fact()
psspy.tysl(0)
#psspy.dyre_new([1,1,1,1],r"""C:\Users\Yang\Documents\PTI\PSSE34\test-IEEE\1-39Bus-test\IEEE39.dyr""","","","")
#psspy.chsb(0,1,[-1,-1,-1,1,1,0])
#psspy.chsb(0,1,[-1,-1,-1,1,12,0])
#psspy.chsb(0,1,[-1,-1,-1,1,13,0])
#psspy.chsb(0,1,[-1,-1,-1,1,16,0])
